cmd/inspr/cli/auth_init.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/inspr/inspr/pkg/cmd/utils"
 )
 
-var authCommand = cmd.NewCmd("cluster").AddSubCommand(
+var clusterCommand = cmd.NewCmd("cluster").AddSubCommand(
 	cmd.NewCmd("init").
 		WithCommonFlags().
 		ExactArgs(1,
```

Comment:
This is about auth. I suggest to change the name of the file to "cluster.go" and add this function in another new file "auth.go" (just like the brokers one).